src/d/actor/d_a_pirate_flag.cpp

```diff
@@ -23,7 +23,7 @@ void daPirate_Flag_packet_c::setNrmVtx(cXyz*, int, int) {
 
 /* 0000083C-00000E44       .text draw__22daPirate_Flag_packet_cFv */
 BOOL daPirate_Flag_packet_c::draw() {
-    /* Nonmatching */
+    return true;
```

Comment:
fyi, we're not perfect at this, but the usual convention is that BOOL goes with TRUE/FALSE, while bool goes with true/false